vslib/inc/saivs.h

```diff
@@ -5,6 +5,10 @@ extern "C" {
 }
 
 #define SAI_KEY_VS_SWITCH_TYPE              "SAI_VS_SWITCH_TYPE"
+#define SAI_KEY_VS_SAI_SWITCH_TYPE          "SAI_VS_SAI_SWITCH_TYPE"
+
+#define SAI_VALUE_SAI_SWITCH_TYPE_NPU       "SAI_SWITCH_TYPE_NPU"
```

Comment:
why you need this ? if i understand correctly this is attribute on switch_create